spec/models/assignment_spec.rb

```diff
@@ -37,6 +37,14 @@
       end.to have_enqueued_job.on_queue("mailers")
     end
 
+    it "sends new assignment notification" do
+      expect do
+        @assignment = FactoryBot.create(:assignment, group: @group, status: "open")
+        NewAssignmentNotification.with(assignment: @assignment).deliver_later(group.first.user)
```

Comment:
Why this?

Reply:
In this one I thought we could test the model function @assignment.notify_recipient .... whenever a new assignment is created .. sends notifications .